content/post/mongodb-arbiter-2022.06.01/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Managing MongoDB with Arbiter on Kubernetes using KubeDB
+date: 2022-06-01
+weight: 20
+authors:
+- Arnob kumar saha
+tags:
+- kubedb
+- MongoDB
+- Arbiter
+- kubernetes
+- replicaset
+- shard
+- cert-manager
+- cost reducing
+- opsRequest
+- verticalScaling
+- Reconfigure
+- enterprise
+- community
+---
+
+## Summary
+
+On 1st June 2022, Appscode held a webinar on **Managing MongoDB with Arbiter on Kubernetes using KubeDB**. Key contents of the webinars are:
+- Introducing the concept of MongoDB Arbiter
+- Importance of voting in MongoDB
+- How Arbiter works on Replica-set database & Sharded cluster.
+- Live Demonstration
+- Q & A Session.
+
+
+
+## Description of the Webinar
+
+Earlier in this webinar, The speaker discussed about the `Arbiter` feature for `MongoDB`. Where He showed how to use kubeDB’s newly introduced features MongoDB-Arbiter. 
```

Comment:
Earlier in this webinar, we discussed about the `Arbiter` feature for `MongoDB`. Where we showed how to use kubeDB’s newly introduced features MongoDB-Arbiter.